mvp-spec/json-schemas/transmission-result.json

```diff
@@ -11,20 +11,39 @@
         "receiver": {
             "$ref": "transmission-receiver.json"
         },
+        "contents": {
+            "type": "array",
+            "items": {
+                "description": "An association of Content-Id with a Transaction Id for building the Audit Log later",
+                "properties": {
+                    "transaction_id": {
+                        "$ref": "transaction_id.json"
+                    },
+                    "content_id": {
+                        "$ref": "transmission-content_id.json"
+                    } 
+                }
+            }
+        },
         "status": {
-            "$ref": "transmission-status.json"
+            "$ref": "transmission-status.json",
+            "description": "Equals \"success\". Transmission Responses with a different status from Suppliers must be dismissed.",
+            "enum": [
+                "success"
+            ]
         },
         "details": {
             "$ref": "transmission-details.json"
         },
         "source": {
-            "description": "Signature based on input:\n```\ntransmission_result.receiver                + '\\u2063' +\ntransmission_result.status                  + '\\u2063' \ntransmission_result.source.domain           + '\\u2063' +\ntransmission_result.source.timestamp        + '\\u2063' +\nseed.source.signature      // -> The Seed associated to the given Transaction Result\n```",
+            "description": "Signature based on input:\n```\ntransmission_response.receiver                + '\\u2063' +\ntransmission_response.status                  + '\\u2063'\ntransmission_response.source.domain           + '\\u2063' +\ntransmission_response.source.timestamp        + '\\u2063' +\nseed.source.signature+ '\\u2063' +\ncontents[0].transaction_ids + '\\u2063' +\ncontents[0].content_id + '\\u2063' +\n... + '\\u2063' +\ncontents[n].transaction_ids + '\\u2063' +\ncontents[n].content_id\n```",
```

Comment:
yes, removing the prefix so that the details describe both Transmission Response and Transmission Result signature. It would be cool if the Json-schema would have an easy way to use inheritance.